pkg/storaged/stratis-details.jsx

```diff
@@ -398,6 +554,56 @@ export const StratisPoolDetails = ({ client, pool }) => {
                         </DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    { pool.Encrypted && client.features.stratis_manage_passphrase &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm className="control-DescriptionListTerm">
+                            {_("storage", "Passphrase")}
+                        </DescriptionListTerm>
+                        <DescriptionListDescription>
+                            { key_desc ? cockpit.format(_("using key description \"$0\""), key_desc) : <i>{_("not configured")}</i> }
+                            <DescriptionListDescription className="tab-row-actions">
+                                { !key_desc
+                                    ? <StorageButton onClick={add_passphrase}>{_("Add")}</StorageButton>
+                                    : null
+                                }
+                                { key_desc
+                                    ? <>
+                                        <StorageButton onClick={change_passphrase}>{_("Change")}</StorageButton>
+                                        { "\n" }
+                                        <StorageButton onClick={remove_passphrase}
+                                                       excuse={!tang_url ? _("This passphrase is the only way to unlock the pool and can not be removed.") : null}>
+                                            {_("Remove")}
+                                        </StorageButton>
+                                    </>
+                                    : null
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)